mage_ai/tests/ai/test_ai_functions.py

```diff
@@ -0,0 +1,133 @@
+import asyncio
+from unittest.mock import MagicMock, Mock
+
+from mage_ai.ai.llm_pipeline_wizard import LLMPipelineWizard
+from mage_ai.data_preparation.models.block import Block
+from mage_ai.data_preparation.models.pipeline import Pipeline
+from mage_ai.tests.base_test import TestCase
+
+EXPECTED_RESPONSE_FOR_LLM_SPLIT = """
+Answer:
+BLOCK 1: function: load data from MySQL. upstream:
+BLOCK 2: function: filter out records column age < 18. upstream: 1
+BLOCK 3: function: export it to postgres. upstream: 2"""
+EXPECTED_LOAD_BLOCK = '{"code": "load mysql"}'
+EXPECTED_FILTER_BLOCK = '{"code": "filter record"}'
+EXPECTED_EXPORT_BLOCK = '{"code": "export postgres"}'
+MOCK_TRANSFORMER_CLASSIFICATION = {
+    "role": "assistant",
```

Comment:
nit: we prefer single quote in the python code.

Reply:
Done